packages/ui-new/src/utils/ThemeConstantsUtil.ts

```diff
@@ -0,0 +1,308 @@
+  semanticWarning060: 'rgba(243, 161, 63, 0.6)',
+  semanticWarning070: 'rgba(243, 161, 63, 0.7)',
+  semanticWarning080: 'rgba(243, 161, 63, 0.8)',
+  semanticWarning090: 'rgba(243, 161, 63, 0.9)',
+  semanticWarning100: 'rgba(243, 161, 63, 1.0)'
+}
+
+export const tokens = {
+  core: {
+    /* Background colors */
+    backgroundAccentPrimary: '#0988F0',
+    backgroundAccentCertified: '#C7B994',
+
+    backgroundWalletKit: '#FFB800',
+    backgroundAppKit: '#FF573B',
+    backgroundCloud: '#0988F0',
+    backgroundDocumentation: '#008847',
+
+    backgroundSuccess: 'rgba(48, 164, 107, 0.20)',
+    backgroundError: 'rgba(223, 74, 52, 0.20)',
+    backgroundWarning: 'rgba(243, 161, 63, 0.20)',
+
+    /* Text colors */
+    textAccentPrimary: '#0988F0',
+    textAccentCertified: '#C7B994',
+
+    textWalletKit: '#FFB800',
+    textAppKit: '#FF573B',
+    textCloud: '#0988F0',
+    textDocumentation: '#008847',
+
+    textSuccess: '#30A46B',
+    textError: '#DF4A34',
+    textWarning: '#F3A13F',
+
+    /* Border colors */
+    borderAccentPrimary: '#0988F0',
+    borderSecondary: '#C7B994',
+
+    borderSuccess: '#30A46B',
+    borderError: '#DF4A34',
+    borderWarning: '#F3A13F',
+
+    /* Foreground colors */
+    foregroundAccent010: 'rgba(9, 136, 240, 0.1)',
+    foregroundAccent020: 'rgba(9, 136, 240, 0.2)',
+    foregroundAccent040: 'rgba(9, 136, 240, 0.4)',
+    foregroundAccent060: 'rgba(9, 136, 240, 0.6)',
+
+    foregroundSecondary020: 'rgba(199, 185, 148, 0.2)',
+    foregroundSecondary040: 'rgba(199, 185, 148, 0.4)',
+    foregroundSecondary060: 'rgba(199, 185, 148, 0.6)',
+
+    /* Icon colors */
+    iconAccentPrimary: '#0988F0',
+    iconAccentCertified: '#C7B994',
+
+    iconSuccess: '#0988F0',
+    iconError: '#DF4A34',
+    iconWarning: '#F3A13F'
+  },
+  dark: {
+    /* Background colors */
+    backgroundPrimary: '#202020',
+    backgroundInvert: '#FFFFFF',
+
+    /* Text colors */
+    textPrimary: '#FFFFFF',
+    textSecondary: '#9A9A9A',
+    textTertiary: '#BBBBBB',
+    textInvert: '#202020',
+
+    /* Border colors */
+    borderPrimary: '#2A2A2A',
+    borderSecondary: '#4F4F4F',
+
+    /* Foreground colors */
+    foregroundPrimary: '#2A2A2A',
+    foregroundSecondary: '#363636',
+    foregroundTertiary: '#4F4F4F',
+
+    /* Icon colors */
+    iconDefault: '#9A9A9A',
+    iconInverse: '#FFFFFF'
+  },
+  light: {
+    /* Background colors */
+    backgroundPrimary: '#202020',
+    backgroundInvert: '#202020',
+
+    /* Text colors */
+    textPrimary: '#202020',
+    textSecondary: '#9A9A9A',
+    textTertiary: '#6C6C6C',
+    textInvert: '#FFFFFF',
+
+    /* Border colors */
+    borderPrimary: '#E9E9E9',
+    borderSecondary: '#D0D0D0',
+
+    /* Foreground colors */
+    foregroundPrimary: '#F3F3F3',
+    foregroundSecondary: '#E9E9E9',
+    foregroundTertiary: '#D0D0D0',
+
+    /* Icon colors */
+    iconDefault: '#9A9A9A',
+    iconInverse: '#202020'
+  }
+}
+
+export const radii = {
```

Comment:
We should give a better name for this instead, just make it `radius` or `borderRadius`?